### PR TITLE
style: apply prettier formatting to md

### LINKED ISSUE #2195: link syntax error
### Describe the bug

https://github.com/search?q=repo%3Ahexojs%2Fsite+%28%28%2F&type=code

error

### Expected behavior

#1607(9047bc4db9b7d081699368819908577b3b1ce899) have fixed this issue in english version.

### PR DESCRIPTION
**Pull request by uiolee**
## Check List

**Please read and check followings before submitting a PR.**

- exec prettier to `index`, `docs/`,`api/` markdown of all languages
- 58a823f4b4799bca550c2758273b3211af3be75f: fix(plugin,themes): fix front-matter 
- 2e6a145ef769ee0da4f2b2215ec50d36277d8539: fix #2195

- [x] Others (Update, fix, translation, etc...)
  - Languages:
  - [ ] `en` English
  - [ ] `ru` Russian
  - [ ] `zh-cn` simplified Chinese
  - [ ] `zh-tw` traditional Chinese

### CODE REVIEW

**Comment by yoshinorin** [comment]
@uiolee 
Would you please tell us which command are using for formatting? We want to apply same formatting to #2150

**Comment by uiolee** [comment]
try `prettier [file] --write` if you want to use CLI.

But I use vscode 'prettier' extension to do this. This way you can directly see the changes

**Comment by yoshinorin** [comment]
@hexojs/core 
May we merge this?

File: package.json [hunk 1]
```diff
@@ -32,13 +33,15 @@
...
+    "lint-staged": "^15.2.0",
+    "prettier": "3.2.5"
   },
   "lint-staged": {
     "*.{png,jpeg,jpg,gif,svg}": [
       "imagemin-lint-staged"
     ],
-    "*.js": "eslint --fix"
+    "*.js": "eslint --fix",
+    "*.md": "format:md"
```
Comment (uiolee) [inline package.json:44]:
```suggestion
    "*.md": "prettier --write"
```

No need to format all files every time

**Comment by yoshinorin** [comment]
Pending #2157

**Comment by D-Sketon** [comment]
waiting #2166

**Review by himself65 — APPROVED** [review]

**Review by yoshinorin — APPROVED** [review]

**Pull request merged**